### PR TITLE
CMake: Reintroduce old installation behavior

### PR DESCRIPTION
**Pull request by tamiko**
As discussed in #509 - reintroduce old relative paths for
the installation of the documentation.

### CODE REVIEW

**Comment by tamiko** [comment]
@guidokanschat: Warning this will also move <code>share/doc/deal.II/html</code> back to <code>doc</code>...

**Comment by tjhei** [comment]
I very much prefer this layout!

**Comment by bangerth** [comment]
Thanks for reverting this part -- I was already afraid that all of my online videos had now become outdated :-)

I don't understand the patch in the 2 minutes I have right now, so I'll let someone else do the review.

File: doc/CMakeLists.txt [hunk 1]
```diff
@@ -51,9 +51,16 @@ ENDIF(DEAL_II_COMPONENT_DOCUMENTATION)
 # Always install a minimalistic README and LICENSE file:
 #
 
+# Bloody hack to get an empty relative path:
```
Comment (guidokanschat) [inline doc/CMakeLists.txt]: You can always add a ``/.`` to a path without changing it

File: cmake/setup_deal_ii.cmake [hunk 2]
```diff
@@ -125,9 +125,12 @@ SET_IF_EMPTY(DEAL_II_LIBRARY_RELDIR "lib${LIB_SUFFIX}")
 SET_IF_EMPTY(DEAL_II_PROJECT_CONFIG_RELDIR "${DEAL_II_LIBRARY_RELDIR}/cmake/${DEAL_II_PROJECT_CONFIG_NAME}")
 SET_IF_EMPTY(DEAL_II_SHARE_RELDIR "share/${DEAL_II_PACKAGE_NAME}")
 SET_IF_EMPTY(DEAL_II_CMAKE_MACROS_RELDIR "${DEAL_II_SHARE_RELDIR}/cmake/macros")
-SET_IF_EMPTY(DEAL_II_DOCREADME_RELDIR "share/doc/${DEAL_II_PACKAGE_NAME}")
-SET_IF_EMPTY(DEAL_II_DOCHTML_RELDIR "${DEAL_II_DOCREADME_RELDIR}/html")
-SET_IF_EMPTY(DEAL_II_EXAMPLES_RELDIR "${DEAL_II_DOCREADME_RELDIR}/examples")
+#
+# ... but install the documentation into prominent places:
+#
```
Comment (guidokanschat) [inline cmake/setup_deal_ii.cmake]: Below is where the ``""`` could be replaced by ``"."``

File: doc/CMakeLists.txt (hunk 1 above)
Comment (tamiko) [inline doc/CMakeLists.txt]: I remember there was a reason for this, i.e. why <code>./</code> was specifically not possible to use. But, let's see.

**Comment by tamiko** [comment]
Simplified as @guidokanschat suggested. Works on *nix.

**Comment by guidokanschat** [comment]
Keep an eye on the online documentation....

**Pull request merged**